styleguide/source/_patterns/01-atoms/forms/search-field-as-in-body.twig

```diff
@@ -0,0 +1,7 @@
+<div class="ama__search__field ama__search__field--in-body">
+  <label for="ama__search__label" class="is-vishidden">{{ searchField.label }}:</label>
+  <input id="{{ searchField.id }}" class="ama__search__field__input" name="{{ searchField.name }}" type="text" placeholder="{{ searchField.placeholder }}" required="" aria-required="true">
```

Comment:
Shouldn't we be using the search field atom here?

Reply:
decision in UX/Dev collab Jun 19 to keep and review this PR as-is to avoid rework and address as tech debt later